ScoutSuite/output/data/html/partials/gcp/services.cloudsql.instances.html

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
It seems like there was a bad merge here too. The `ScoutSuite/output/data/html/partials/gcp/services.cloudsql.projects.id.instances.html` file was deleted, but I it's the most recent one.